image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl

```diff
@@ -76,12 +76,12 @@
 {{- end -}}
 
 {{/* Configure images, certificates and passwords as required. */}}
-{{/* Note that for secured-cluster-services we don't configure certificates here, instead they will be distributed at runtime by Sensor and Central. */}}
+{{/* Note that for secured-cluster-services we usually don't configure certificates here, instead they will be distributed at runtime by Sensor and Central. */}}
```

Comment:
You are right, this seems confusing. I think the reason why I added "usually" here is because for some of the standard tests contained in the helmtest test suite for secured-cluster-services the ability to inject certificates is quite important. I will add a note to the comment which hopefully clears it up.